snowflake-schema-grant-all/main.tf.json

```diff
@@ -39,6 +39,7 @@
   },
   "locals": {
     "privileges": [
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>local.privileges is declared but not used